src/pycrunch/elements.py

```diff
@@ -323,7 +327,7 @@ def email(self):
             PendingDeprecationWarning,
         )
         if self.__email is None:
-            self.__email = self.user["body"]["email"]
+            self.__email = self.get(self.site_url).payload.user["body"]["email"]
```

Comment:
Suggestion: Add unit test for this property

Reply:
Done in https://github.com/Crunch-io/pycrunch/pull/78/commits/ef4edd53850f9456f3c65a43154b32727c3416c4